website/docs/r/vapp_vm.html.markdown

```diff
@@ -246,6 +246,34 @@ resource "vcd_vapp_vm" "secondVM" {
 
 ```
 
+## Example Usage (using advanced compute settings)
+This example shows how to create an empty VM with advanced compute settings.
+
+```hcl
+resource "vcd_vapp_vm" "advancedVM" {
+  vapp_name     = vcd_vapp.web.name
+  name          = "advancedVM"
+  computer_name = "advancedVM"
+  memory        = 2048
+  cpus          = 2
+  cpu_cores     = 1
+
+  os_type          = "sles10_64Guest"
+  hardware_version = "vmx-14"
+  catalog_name     = "my-catalog"
+  boot_image       = "myMedia"
+
+  memory_priority_type = "CUSTOM"
+  memory_shares        = "480"
+  memory_reservation   = "8"
+  memory_limit         = "48"
+
+  cpu_priority_type = "CUSTOM"
```

Comment:
Same thing as above. `cpu_priority_type` doesn't exist, it is called `cpu_priority`.
```suggestion
  cpu_priority = "CUSTOM"
```

Reply:
done